js/main.js

```diff
@@ -0,0 +1,196 @@
+'use strict'
+
+var Node = function(data){
+  this.data = data;
+  this.next = null;
+  this.previous = null;
+}
+
+var List = function(){
+  this.head = null;
+  this.tail = null;
+  this.length = 0;
+}
+
+List.prototype.getHead = function(){
+  return this.head;
+}
+
+List.prototype.getTail = function(){
+  return this.tail;
+}
+
+List.prototype.append = function(data){
+  var node = new Node(data);
+
+  if(this.length){
+    this.tail.next = node;
+    node.previous = this.tail;
+    this.tail = node;
+  }
+  else{
+    this.head = node;
+    this.tail = node;
+  }
+
+  this.length++;
+
+  return node;
+}
+
+List.prototype.prepend = function(data){
+  var node = new Node(data);
+
+  if(this.length){
+    this.head.previous = node;
+    node.next = this.head;
+    this.head = node;
+  }
+  else{
+    this.head = node;
+    this.tail = node;
+  }
+
+  this.length++;
+
+  return node;
+}
+
+List.prototype.checkRange = function(position){
+  if(position > this.length || position < 0){
+    throw new RangeError("Uncorrect position in 'at' function");
```

Comment:
Not only 'at' function can throw this exception. I think the error message is incorrect